pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -522,10 +522,6 @@ func (d *DependenciesDistributor) createOrUpdateAttachedBinding(attachedBinding
 	key := client.ObjectKeyFromObject(attachedBinding)
 	err := d.Client.Get(context.TODO(), key, existBinding)
 	if err == nil {
-		if util.GetLabelValue(existBinding.Labels, workv1alpha2.ResourceBindingPermanentIDLabel) == "" {
-			existBinding.Labels = util.DedupeAndMergeLabels(existBinding.Labels,
-				map[string]string{workv1alpha2.ResourceBindingPermanentIDLabel: uuid.New().String()})
-		}
```

Comment:
Why remove this? Can we ensure that the `existBinding` has the label?

Reply:
yes, it can only be created here.
```go
attachedBinding.Labels = util.DedupeAndMergeLabels(attachedBinding.Labels,
		map[string]string{workv1alpha2.ResourceBindingPermanentIDLabel: uuid.New().String()})
if err := d.Client.Create(context.TODO(), attachedBinding); err != nil {
```

Reply:
get it.